io/util.ts

```diff
@@ -20,6 +20,18 @@ export function charCode(s: string): number {
 
 const encoder = new TextEncoder();
 export function stringsReader(s: string): Reader {
-  const ui8 = encoder.encode(s);
-  return new Buffer(ui8.buffer as ArrayBuffer);
+  return new Buffer(encoder.encode(s).buffer);
+}
+
+export async function tempFile(
+  dir: string,
+  opts: {
+    prefix?: string;
+    postfix?: string;
+  } = { prefix: "", postfix: "" }
+): Promise<{ file: File; filepath: string }> {
+  const r = ~~(Math.random() * 1000000);
```

Comment:
feels a bit strange to me... the double bitwise operator... at the least it is unclear... while I am not rejecting it, it does feel like it temporary files are something that Deno itself should be handling.  Especially because there is no contract here to purge it.

@ry it feels like a tempfile is something that runtime users should be abstracted from.

Reply:
@keroxp I agree that the double `~` needs a comment - I don't know what that's doing.

This feels like a function that can be moved into a common utility module at some point (or even into core) but I think for now let's just land it here.

Reply:
I believe it gets rid of decimal part of number, ie. `~~1.23456 === 1`

Reply:
I see. I'd prefer if @keroxp uses Math.floor() then as it's more readable.